### ISSUE TITLE
Multiple ``span`` typing position confused

### ISSUE THREAD

**Issue by iamanewofkotlin** [alexmacarthur/typeit#267]
I have a text below. It's generated by ``prisma.js``, and i promising its valid.

```
const text = `  <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>table</span> <span class="token attr-name">:fooa</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">&quot;</span>bara<span class="token punctuation">&quot;</span></span> <span class="token attr-name">:foob</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">&quot;</span>barb<span class="token punctuation">&quot;</span></span> <span class="token punctuation">/&gt;</span></span>`;
```

my expected is: ``  <table :fooa="bara" :foob="barb" /> ``

typed result is: ``  <table :fooa="bara"=""/> :foobbarb ``

![Here is print screen](https://user-images.githubusercontent.com/51434344/144186650-9abf6dce-dda0-4704-9bed-a866455d53e5.gif)

Here is formatted code
```
<span class="token tag">
	<span class="token tag">
		<span class="token punctuation">
			&lt;
		</span>
		table
	</span>
	<span class="token attr-name">
		:fooa
	</span>
	<span class="token attr-value">
		<span class="token punctuation attr-equals">
			=
		</span>
		<span class="token punctuation">
			&quot;
		</span>
		bara
		<span class="token punctuation">
			&quot;
		</span>
	</span>
	<span class="token attr-name">
		:foob
	</span>
	<span class="token attr-value">
		<span class="token punctuation attr-equals">
			=
		</span>
		<span class="token punctuation">
			&quot;
		</span>
		barb
		<span class="token punctuation">
			&quot;
		</span>
	</span>
	<span class="token punctuation">
		/&gt;
	</span>
</span>
```

**Comment by alexmacarthur** [comment]
Ah... hang tight on this. I know what's going on, but trying to think through what the best way to fix it is. It may involve some pretty intense (but very welcome) changes to how strings are parsed.

**Comment by iamanewofkotlin** [comment]
It maybe isn't a common feature for ``typeit``, please don't mind sir. I had been tried to resolving the issue, but i can't to do.
The ``node`` looks like is parsed by ``native html``. That means ``TypeIt`` had done it's best so far.

**Comment by alexmacarthur** [comment]
@iamanewofkotlin Fixed in v8.2.0! This turned out to be a great opportunity to revamp how the library parses HTML and prints it to the screen. Figured out a way to do it more reliably, efficiently, and simply, so thanks for reporting. It also cuts the library's size by 5-7%. 👍 

https://github.com/alexmacarthur/typeit/releases/tag/v8.2.0
